api/v1/controllers/collectors.js

```diff
@@ -37,8 +35,23 @@ const ZERO = 0;
  * @param {Function} next - The next middleware function in the stack
  */
 function postCollector(req, res, next) {
-  // TODO reject if caller is not a collector
-  doPost(req, res, next, helper);
+  const collectorToPost = req.swagger.params.queryBody.value;
```

Comment:
Pls. add a comment describing the extra work being done here, e.g. "When a collector registers itself with Refocus, Refocus sends back a special token for that collector to use for all further communication."